README.md

```diff
@@ -1,30 +1,24 @@
-# Pants Build System
+# Pantsbuild
 
-Pants is a scalable build system for _monorepos_: codebases containing 
-multiple projects, often using multiple programming languages and frameworks, 
-in a single unified code repository.
+Pantsbuild is a fast, scalable, user-friendly build system for codebases of 
```

Comment:
I would consider leaving in a few references to "Pants" to disambiguate (e.g. "Pantsbuild (also known as Pants)")

Reply:
I was thinking about something like "Pantsbuild (aka Pants)" but didn't know if it's too distracting. Any suggestions for where to put it?